packages/contentful--react-apps-toolkit/package.json

```diff
@@ -46,6 +46,8 @@
     "access": "public"
   },
   "dependencies": {
-    "@contentful/app-sdk": "^4.0.0"
+    "@contentful/app-sdk": "^4.0.0",
+    "contentful-management": "^8.2.0",
+    "react": "^17.0.2"
```

Comment:
`react` is a peer dependency. Please install it as a dev dependency as well.

Reply:
I believe this is fixed now.